apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SessionPool.cs

```diff
@@ -205,10 +262,24 @@ public PooledSession CreateDetachedSession(SessionName sessionName, ByteString t
         /// <param name="databaseName">The database whose session pool should be populated. Must not be null.</param>
         /// <param name="cancellationToken">An optional token for canceling the call.</param>
         /// <returns>A task which will complete when the session pool has reached its minimum size.</returns>
-        public Task WhenPoolReady(DatabaseName databaseName, CancellationToken cancellationToken = default)
+        public Task WhenPoolReady(DatabaseName databaseName, CancellationToken cancellationToken = default) =>
+            WhenPoolReady(databaseName, null, cancellationToken);
+
+        /// <summary>
+        /// Returns a task indicating when the session pool associated with the given database name is populated up to its minimum size.
```

Comment:
We'll need to update the comment here - presumably to "with the given key" or similar. If we keep this precise signature, then the comment should be "with the given database name and role".